sdk/core/azure-core-amqp/test/ut/message_sender_receiver.cpp

```diff
@@ -20,853 +20,882 @@
 
 #include <gtest/gtest.h>
 
-extern uint16_t FindAvailableSocket();
+namespace Azure { namespace Core { namespace Amqp { namespace Tests {
```

Comment:
should the file be called `message_sender_receiver_tests.cpp`?

Reply:
I'm generally not a fan of putting _tests in filenames in the Tests directory. It is redundant. Similarly, I'm not a big fan of writing "FooTest" in a test file - again, it's redundant.

Reply:
other libs have it